src/sdk/formatting/formatNumber.ts

```diff
@@ -12,7 +12,7 @@ export type NumberFormatOptions = {
 	unit?: string;
 };
 
-export const formatNumber = (number: number, options?: NumberFormatOptions): string => {
+export const formatNumber = (number: number, options?: NumberFormatOptions, formatter: Intl.NumberFormat = numberFormatter): string => {
```

Comment:
I don't think that `formatter` should be publicly exposed here. Since `options` is always being passed in from `formatCurrency` as `null`, there really is no value in leveraging this method from `formatCurrency` and it could be simplified to
```js
if (!Boolean(number) || isNaN(number)) {
	number = 0;
}
return formatter.format(number);
```

Reply:
@smbecker Made these changes [here](https://github.com/portalarchitects/skysync-cli/pull/220)